README.md

```diff
@@ -1,488 +1,57 @@
-export IGN_DISABLE_SUMMARY_EMAILS=false
-export IGN_DEFAULT_EMAIL_RECIPIENT=
-export IGN_DEFAULT_EMAIL_SENDER=cloudsim@osrfoundation.org
-
-# Logging
-export IGN_LOGGER_LOG_STDOUT=true
-# Verbosity - 4 debug, 3 info, 2 warning, 1 error, 0 critical
-export IGN_LOGGER_VERBOSITY=4
-export IGN_DB_LOG=false
-
-# DO NOT log to Rollbar
-export IGN_ROLLBAR_TOKEN=
-export IGN_ROLLBAR_ENV=
-export IGN_ROLLBAR_ROOT=
-
-# EC2 Machines
-export IGN_EC2_AVAILABILITY_ZONES=us-east-1a
-export IGN_EC2_MACHINES_LIMIT=-1
-# All of the following variables should be updated to match the corresponding ids in your AWS account
-export IGN_EC2_AMI=ami-08861f7e7b409ed0c
-export IGN_EC2_SECURITY_GROUPS=sg-0c5c791266694a3ca
-export IGN_EC2_SUBNETS=subnet-0e632d68a9032ab9d
-
-# Configure AWS access
-export AWS_ACCOUNT=your account id (eg. used by AWS ECR)
-export AWS_REGION=us-east-1
-export AWS_ACCESS_KEY_ID=your access key id
-export AWS_SECRET_ACCESS_KEY=your secret access key
-
-# Auth0
-export AUTH0_RSA256_PUBLIC_KEY=xxxxxxxxxxxxxxxxxxx
-```
-
-# User registration
-
-An authorized user is required to perform operations. Users are planned to be 
-stored in a common database shared by all services. For now, registered 
-users are currently being stored in the `fuel` database setup by 
-`ign-fuelserver`. 
-
-The following steps create a user in the `fuel` database. 
-This step requires a running instance of 
-[ign-fuelserver](https://bitbucket.org/ignitionrobotics/ign-fuelserver) 
-and [web-app](https://bitbucket.org/ignitionrobotics/web-app). If the user is
- required for local development, then these instances should be local as well.
-
-1. (Local development) Start `ign-fuelserver`.
-1. (Local development) Start `web-app`.
-1. Access the `ign-fuelserver` UI.
-1. Login with an Auth0 account.
-1. Insert your username to create a user and assign it the logged in Auth0 
-account.
-
-# Example routes to submit and shutdown simulations
-
-Submit a new simulation:
-```
-curl -k -X POST --url http://localhost:8001/1.0/simulations -F name=testSim --header 'authorization: Bearer <token>'
-```
-
-Delete an existing simulation:
-```
-curl -k -X DELETE --url http://localhost:8001/1.0/simulations/{simulation-groupID} --header 'authorization: Bearer <token>'
-```
-
-Tip: an easy way to test if the web-cloudsim can connect to a kubernetes cluster is to run:
-```
-curl http://localhost:8001/1.0/k8/countpods --header 'authorization: Bearer <token>'
-```
-
-Note: To obtain `<token>` for a user, login with the user's Auth0 account on 
-your target `ign-fuelserver` instance. The JWT bearer token can be found in the 
-`token` value of the Local Storage.
+  <img src="./assets/logo.png" width="200" alt="Gazebo" />
+  <h1>Gazebo Cloudsim</h1>
+  <p>Cloudsim is an open source library for creating robotic applications in the cloud. It allows to create applications that run Gazebo simulations in Kubernetes. It currently has support for AWS, but support for other providers can be implemented.</p>
```

Comment:
I feel like robotic applications is a bit broad. It should be "run robotic simulations in the cloud" (hence the name Cloudsim) or "run robotic applications in the cloud" if you want to go with something more generic.

```suggestion
  <p>Cloudsim is an open source library to run robotic applications in the cloud. It currently supports deployments using Kubernetes and AWS resources, but support for other providers can be implemented.</p>
```